docs/user/tutorial/60-admin.rst

```diff
@@ -18,23 +18,25 @@ Using the Filer in the Django Admin
 
 In the Toolbar, select the *Site menu* and choose *Administration*.
 
-.. todo:: update for 3.2
+.. todo:: update for 3.2. -- 3.2 expands admin automatically
 
 .. image:: images/zoom_panel.png
    :alt: The zoom button
    :align: right
-   :width: 10%
+   :width: 50%
 
 This opens the django CMS *overlay*, which provides a quick view onto some extra control
 functionality.
 
-.. todo:: screenshot of the overlay
+.. image:: images/zoom_panel.png
```

Comment:
There is a lot of white space in the image of the overlay - would it be possible to make the whole browser window narrower, and re-take the image at the minimum width possible so its contents are clearer?